server/api/products.js

```diff
@@ -0,0 +1,62 @@
+const Sequelize = require('sequelize');
+const router = require('express').Router()
+const { Product } = require('../../db')
+
+router.get('/', async function (req, res, next){
+	try {
+		const products = await Product.findAll()
+  		res.status(200).send(products)
+	} catch (error) {
+		console.log(error)
+		next(error)
+	}
+});
+
+router.get('/:productId', async function (req, res, next){
+	try {
+		const product = await Product.findOne({
+			where: {
+			  id: req.params.productId
+			},
+		})
+  		res.status(200).send(product)
```

Comment:
If findOne() doesn't find anything, it doesn't actually throw an error.
You should check for and handle the case where product is empty

Reply:
Incorporated